### PR TITLE
response modification updated

### PR DESCRIPTION
**Pull request by szucst**

### CODE REVIEW

File: WebdriverIO/tests/response-modification.spec.ts [hunk 1]
```diff
@@ -32,31 +36,33 @@ describe('Response modification', () => {
         expectedURL,
         expectedResponseStatusCode,
       );
-      await browser.assertExpectedRequestsOnly();
+      await browser.expectRequest(
```
Comment (mgabor18) [inline WebdriverIO/tests/response-modification.spec.ts]: Why do you call this `expectRequest` method twice?

**Review by mgabor18 — CHANGES_REQUESTED** [review]

File: WebdriverIO/tests/response-modification.spec.ts (hunk 1 above)
Comment (szucst) [reply WebdriverIO/tests/response-modification.spec.ts]: They are called with different status codes: `expectedResponseStatusCode` and `modifiedResponseStatusCode`. And after that is called the `assertRequests()` which compares the expectations.

Comment (mgabor18) [reply WebdriverIO/tests/response-modification.spec.ts]: Then my question is, why is the first one called? :D
My way of thinking is clicking only once, and validating that modified response.

Comment (szucst) [reply WebdriverIO/tests/response-modification.spec.ts]: On second thought, I agree with you. I see no point in calling the first one. I modified it. Thanks :)

**Review by mgabor18 — APPROVED** [review]

**Review by sorosz89 — APPROVED** [review]

**Pull request merged**